pkg/fanal/applier/applier_test.go

```diff
@@ -961,6 +961,91 @@ func TestApplier_ApplyLayers(t *testing.T) {
 			},
 			wantErr: "unknown OS",
 		},
+		{
+			name: "SUSE Images - Legacy OS Name with Backward Compatibility",
```

Comment:
nit: Other cases are lowercase.

```suggestion
			name: "SUSE images - legacy OS name with backward compatibility",
```